docs/pages/application-access/getting-started.mdx

```diff
@@ -4,11 +4,24 @@ description: Getting started with Teleport application access.
 videoBanner: 5Uwhp3IQMHY
 ---
 
-Let's connect to Grafana using Teleport in three steps:
+This tutorial demonstrates how to configure secure access to an application through Teleport. 
+The tutorial uses Grafana as a sample application because it's straightforward to install and run 
+in a Docker container or Kubernetes cluster with no additional configuration required.
+If you want to configure access for a different web application, you can use this tutorial as a 
+general guide for what to do. 
```

Comment:
I don't think we've documented it anywhere, but we put hard line breaks in the docs around 80 characters, to make viewing/editing the source easy regardless of the editor being used:

```suggestion
This tutorial demonstrates how to configure secure access to an application
through Teleport. The tutorial uses Grafana as a sample application because
it's straightforward to install and run in a Docker container or Kubernetes
cluster with no additional configuration required. If you want to configure
access for a different web application, you can use this tutorial as a general
guide for what to do. 
```

This comment applies throughout.

Reply:
Yeah, I see. I generally have followed the one sentence per line rule instead. But I'm flexible.